app/views/pages/home.html.erb

```diff
@@ -5,27 +5,22 @@
       <li class="valign-wrapper">
         <%= image_tag 'gi-bill.jpg' %>
         <div class="caption center-align valign">
-          <h3>
-            Welcome back. You served your country.<br />
-            Now let's get you coding and to your next mission.
-          </h3>
+          <h3>Thank you for serving</h4>/>
+          <h4>Now let's get you coding and on to your next mission</h4>
```

Comment:
I don't care either way but it should be consistent. @dengjonathan can you go through and make sure each item either has punctuation or doesn't?